doc/api/crypto.md

```diff
@@ -2949,6 +2949,160 @@ Does not perform any other validation checks on the certificate.
 
 ## `node:crypto` module methods and properties
 
+### `crypto.argon2(password, salt, keylen[, options], callback)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `password` {string|ArrayBuffer|Buffer|TypedArray|DataView}
+* `salt` {string|ArrayBuffer|Buffer|TypedArray|DataView}
+* `keylen` {number}
```

Comment:
I can't remember, with argon are there constraints on the keylen? min length, max length, etc.

Reply:
Let's be sure to include the constraints in the documentation here.